python/cuml/experimental/explainer/common.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cupy as cp
```

Comment:
Tests of this file are in the kernel shap PR

Reply:
Indeed it is the same, I just didn't merge them to not conflate both reviews, but after solving these set of comments I can merge this into the kernel shap PR, what do you think?